autotest/gdrivers/rda.py

```diff
@@ -458,6 +458,54 @@ def rda_graph_nominal():
 
     ds = None
 
+    # File open
+    with webserver.install_http_handler(handler):
+        with gdaltest.config_options(config_options):
+            with open('/tmp/rda_test1.dgrda', "w") as fd:
```

Comment:
Use tmp/rda_test1.dgrda to avoid issues on Windows.

Reply:
done